app/create_file.py

```diff
@@ -1 +1,41 @@
-# write your code here
+import os
+import sys
+from datetime import datetime
+
+
+def create_directory(console_data):
+    parent_dir = console_data[console_data.index("-d") + 1]
+    path = os.path.join(parent_dir, console_data[console_data.index("-d") + 2])
```

Comment:
why `+2`, if, for example, we want to create 5 or 10 folders
```suggestion
    path = os.path.join(parent_dir, console_data[console_data.index("-d") + ???])
```